### PR TITLE
:gem:  Preserve data :gemini: 

### PR DESCRIPTION
**Pull request by SimonGideon**
## This PR contains the following:

1. Store data into respective `json` file. The files are created if doesn't exist.
2. On exit from your app save all the data (books, people, and rentals) into JSON files. One per type of data you are storing and named accordingly

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -14,6 +15,13 @@ def initialize()
     @rentals = []
   end
```
Comment (EricMbouwe) [inline app.rb:17]: - [ ] On startup the app loads all data from files in the `initialize` method. Let's fix that
Ex: `@books = load_books()`

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
Hi @SimonGideon, 

Good job so far 🔥!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🔥🎉
- Preserve data ✔
- No linter errors ✔

# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 1 above)
Comment (SimonGideon) [reply app.rb:17]: @EricMbouwe . I'm I supposed to get rid of the instance variable array from the initialize method ? :thinking: :thought_balloon:

**Review by Owusu-Desmond — APPROVED** [review]
Hi @SimonGideon , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by SimonGideon** [comment]
@Owusu-Desmond Thanks for the review :slightly_smiling_face:

**Pull request merged**